### PR TITLE
add bsc chain

### PR DESCRIPTION
**Pull request by ewilz**

### CODE REVIEW

**Review by TDG510 — COMMENTED** [review]
Ok thAnks

**Review by TDG510 — COMMENTED** [review]
Ok thAnks

**Review by TDG510 — COMMENTED** [review]
Ok thAnks!!!!

**Comment by wils81ouwehand** [comment]
afdc65f3d5328701a7b426733366aef5ede9d99f

**Comment by StemmlerSisters** [comment]
### __

**Comment by Jamstar00** [comment]
> _ No description provided. _

bsc

**Review by Sturgeontara06 — COMMENTED** [review]
How withdrawal

**Review by Ciornenichii — COMMENTED** [review]
Ciornenichii stepan bilan

**Review by Sturgeontara06 — COMMENTED** [review]
Tgank uou

**Review by Joj501 — APPROVED** [review]

**Review by Sturgeontara06 — COMMENTED** [review]
K

**Review by Reallyfe — COMMENTED** [review]
Commit

**Review by Scutua — APPROVED** [review]

**Review by Scutua — APPROVED** [review]

**Review by Reallyfe — COMMENTED** [review]
App

File: hardhat.config.ts
Comment (Tq101510) [inline hardhat.config.ts:1]: [0420868990.csv](https://github.com/user-attachments/files/22017275/0420868990.csv)

**Review by Eusinho1985 — APPROVED** [review]

**Review by Christymulligan7 — APPROVED** [review]

**Pull request merged**